server/scratchy_server/resources/messageres.py

```diff
@@ -20,6 +20,7 @@ def post(self, **kwargs):
     def put(self, messageId, **kwargs):
         message = MessageModel.objects().get_or_404(id=messageId)
         message.update(**kwargs)
+        message = MessageModel.objects().get_or_404(id=messageId)
```

Comment:
Hum why did you do that ? it seems strange just reading the code, did you take a look at the documentation, especially [modify method](https://docs.mongoengine.org/apireference.html#mongoengine.Document.modify) that seems to do this job in a single call : 
```
Perform an atomic update of the document in the database and reload the document object using updated version.

Returns True if the document has been updated or False if the document in the database doesn’t match the query.
```

Update VS modify : https://github.com/MongoEngine/mongoengine/issues/2060#issuecomment-522508301

Reply:
I can see it's done :+1: